.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -0,0 +1,16 @@
+name: publish source-deploy-retrieve-bundle
+on:
+  workflow_dispatch:
```

Comment:
so you're going to manually push the button every time, or is some other automated process is going to wait for it?

Suggestion: maybe this should run after each time something else runs (ex: after a gh release publishes, or after the normal npm publish action runs) so that you have on for every version?

Or is this just temporary and you'll automate more later?